java/performance_test/src/main/java/io/ray/performancetest/TestUtils.java

```diff
@@ -0,0 +1,15 @@
+package io.ray.performancetest;
+
+public class TestUtils {
+  private static boolean DEV_MODE = false;
+
+  static {
+    if (null != System.getenv("PERF_TEST_DEV")) {
+      DEV_MODE = true;
+    }
+  }
+
+  public static boolean isDevMode() {
```

Comment:
We don't need this in the community version, right?

Reply:
removed.